### ISSUE TITLE
[Android] Improve TalkBack announcement of S-meter 

### ISSUE THREAD

**Issue by VanceVagell** [VanceVagell/kv4p-ht#160]
The S-meter should have a programmatically updated accessibility label that reads out the S-meter value when a blind / low vision person is using the app with TalkBack. Also, only the S-meter container UI should receive focus, not the individual bars in the graphic.
